src/main/java/ticket/TicketService.java

```diff
@@ -47,5 +49,25 @@ public static void main(String[] args) {
         }
         users.delete(client);
         System.out.println(users.contain(client));
+
+        TicketDao ticketDao = new TicketDao();
+        UserDao userDao = new UserDao();
+        Ticket ticket1 = new Ticket();
+        ticket1.setId(1);
+        ticket1.setTicketType(TicketType.MONTH);
+        Ticket ticket2 = new Ticket();
+        ticket2.setId(2);
+        ticket2.setTicketType(TicketType.MONTH);
+        User user = new User();
+        user.setId(1);
+        user.setName("Alex");
+        userDao.saveUser(user);
+        ticketDao.saveTicket(ticket1, user.getId());
+        ticketDao.saveTicket(ticket2, user.getId());
+        System.out.println(ticketDao.fetchTicketById(1));
+        for (Ticket t : ticketDao.fetchTicketByUserId(user.getId())) {
+            System.out.println(t);
+        }
+        ticketDao.updateTicketType(1, TicketType.YEAR); // Can't figure why it doesn't work
```

Comment:
see answer in comment in TicketDao class

Reply:
The answer lies on the surface, as usual)))